controllers/front/listing/SearchController.php

```diff
@@ -66,7 +66,7 @@ protected function getProductSearchQuery()
     {
         $query = new ProductSearchQuery();
         $query
-            ->setSortOrder(new SortOrder('product', Tools::getProductsOrder('by'), Tools::getProductsOrder('way')))
+            ->setSortOrder(new SortOrder('product', Tools::getProductsOrder('by'), 'desc'))
```

Comment:
It seems you are removing the sorting feature from the search by forcing a value. Are you sure about this change?

Reply:
Yes you are correct.  That file was not suppose to make it into the commit.  This is what I get for rushing.  I'll amend this commit to remove this change.  My apologies.